Harvest.Web/ClientApp/src/Tickets/TicketTable.tsx

```diff
@@ -0,0 +1,54 @@
+import { useMemo } from "react";
+import { Cell, Column, TableState } from "react-table";
+import { ReactTable } from "../Shared/ReactTable";
+import { ReactTableUtil } from "../Shared/TableUtil";
+import { Ticket } from "../types";
+
+interface Props {
+  tickets: Ticket[];
+}
+
+export const TicketTable = (props: Props) => {
+  const ticketData = useMemo(() => props.tickets, [props.tickets]);
+  const columns: Column<Ticket>[] = useMemo(
+      () => [
+          {
+              Cell: (data: Cell<Ticket>) => (
+                  <div>
+                      <a href={`/ticket/details??projectId=${data.row.original.projectId}&id=${data.row.original.id}`}>
```

Comment:
need to use the `<Link>` not the `<a>` if it's a react route

Reply:
@srkirkland I based this off the invoice table. Want me to update that one too?

Reply:
@jSylvestre maybe but I have a PR open which changes the invoices table a lot so be careful, or do it later.